include/verilated_vpi.cpp

```diff
@@ -2604,6 +2652,42 @@ void vl_vpi_get_value(const VerilatedVpioVarBase* vop, p_vpi_value valuep) {
 
     const int varBits = vop->bitSize();
 
+    // TODO: __VforceRd already has the correct value, but that signal is not public and thus not
```

Comment:
```suggestion
    // __VforceRd already has the correct value, but that signal is not public and thus not
```

Presume this now works...

Reply:
Well, no. That's what #6705 is for. I edited the comment to make this clearer.